server/config/test.exs

```diff
@@ -36,7 +36,9 @@ config :tuist, Tuist.Repo,
   hostname: "localhost",
   database: "tuist_test#{System.get_env("MIX_TEST_PARTITION")}",
   pool: Ecto.Adapters.SQL.Sandbox,
-  pool_size: System.schedulers_online()
+  pool_size: System.schedulers_online() * 2,
+  queue_target: 5000,
+  queue_interval: 1000
```

Comment:
Since many tests run concurrently, and many create and read organizations, with the encrypting/decrypting of the organization operations are taking a bit longer to execute, causing some connections from tests to drop. This shouldn't be an issue in production where we don't preload organizations in highly-concurrent endpoints, nor have many organizations being created.